src/site/_includes/codelab.njk

```diff
@@ -1,98 +1,91 @@
+            <time>{{date | prettyDate}}</time>
+            {% if updated %} — {{ 'i18n.common.updated' | i18n((locale)) }} <time>{{ updated | prettyDate }}</time> {% endif %}
+          </div>
+        {% endif %}
 
-          {% if authors %}
-            <div class="cluster flow-space-size-2">
-              {% for author in authors %}
-                {% Author {
-                  id: authorKey,
-                  author: collections.authors[author],
-                  showSocialMedia: true,
-                  locale: locale
-                  }
-                %}
-              {% endfor %}
-            </div>
-          {% endif %}
-        </header>
+        {% SignPosts page.fileSlug %}
 
-        {% if draft and site.env === 'dev' %}
-          <div class="banner bg-state-warn-bg color-core-text">
-            <p><strong>{{ 'i18n.post.is_draft' | i18n(locale) }}</strong></p>
+        {% if authors %}
+          <div class="cluster flow-space-size-2">
+            {% for author in authors %}
+              {% Author {
+                id: authorKey,
+                author: collections.authors[author],
+                showSocialMedia: true,
+                locale: locale
+                }
+              %}
+            {% endfor %}
           </div>
         {% endif %}
+      </header>
 
-        {{ content | safe }}
-
-        <div class="text-size-0 color-mid-text">
-          <span>
-            {% if updated %}
-            {{ 'i18n.post.last_updated' | i18n(locale) }}: <time>{{ updated | prettyDate }}</time>
-            {% else %}
-            {{ 'i18n.post.last_updated' | i18n(locale) }}: <time>{{ date | prettyDate }}</time>
-            {% endif %}
-          </span>
-          —
-          <a
-            href="{{ page.inputPath | githubLink }}"
-          >
-            {{ 'i18n.post.improve_article' | i18n(locale) }}
-          </a>
+      {% if draft and site.env === 'dev' %}
+        <div class="banner bg-state-warn-bg color-core-text">
+          <p><strong>{{ 'i18n.post.is_draft' | i18n(locale) }}</strong></p>
         </div>
+      {% endif %}
 
-        {% if related_post %}
-          {% set backUrl = '/' + related_post %}
-          {% set backLabel = 'Return to article' %}
-        {% elif path %}
-          {% set backUrl = '/' + path.slug %}
-          {% set backLabel = 'Return to all articles' %}
-        {% endif %}
+      {{ content | safe }}
 
-        {% if backLabel and backUrl %}
-          <div class="flow-space-size-2">
-            <a href="{{ backUrl }}" class="button" data-type="secondary">
-              {% include 'icons/arrow-back.svg' %}
-              <span>{{ backLabel }}</span>
-            </a>
-          </div>
-        {% endif %}
-      </article>
-    </div>
-  </web-codelab>
-</div>
+      <div class="text-size-0 color-mid-text">
+        <span>
+          {% if updated %}
+          {{ 'i18n.post.last_updated' | i18n(locale) }}: <time>{{ updated | prettyDate }}</time>
```

Comment:
Nit: I think this line (and 62) should be indented.

Reply:
Totally agree, I'll fix this now.